test/acceptance/keywords/lists.robot

```diff
@@ -148,6 +148,18 @@ List Should Have No Selections
     ...    List 'interests' should have had no selection (selection was [ Males | Females | Others ])
     ...    List Should Have No Selections    interests
 
+Get List Values From Single-Select List
```

Comment:
Perhaps it would be good to place the test before the `Get Selected List Value`, then the test related to same keyword are in same location.